app/templates/pdf/attendees_pdf.html

```diff
@@ -123,7 +123,7 @@ <h3 style="text-align:center;">{{ holders[0].event.name }} - {{ ("Attendees List
                         {{ holder.email }}<br>
                     </strong>
                     {% if holder.ticket.price %}
-                        {{ holder.event.payment_currency | currency_symbol }}{{ holder.ticket.price | money }}
+                        {{ holder.event.payment_currency }}{{ holder.ticket.price | money }}
```

Comment:
@kushthedude please add screenshot of the change for reference.

Reply:
## Problem
<img width="311" alt="Screenshot 2019-09-02 at 7 49 57 AM" src="https://user-images.githubusercontent.com/44091822/64086503-5648bd80-cd56-11e9-9e61-bf8abe4627c3.png">

## Now
<img width="603" alt="Screenshot 2019-09-02 at 7 49 47 AM" src="https://user-images.githubusercontent.com/44091822/64086510-59dc4480-cd56-11e9-9e08-6049cb476b96.png">